homeassistant/components/iotawatt/config_flow.py

```diff
@@ -0,0 +1,100 @@
+"""Config flow for iotawatt integration."""
+import json
+import logging
+
+import httpx
+from httpx import AsyncClient
+from iotawattpy.iotawatt import Iotawatt
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): str,
```

Comment:
Let's not ask for a name and just use the host as the default name.

Reply:
@balloob I removed it then remembered an issue that was brought up when a user has more than 1 IoTaWatt device and the sensor names are the same; [https://github.com/gtdiehl/iotawatt_ha/issues/14](https://github.com/gtdiehl/iotawatt_ha/issues/14).

Currently the sensor name is a concatenation of ```iotawatt``` + ```I/O Type``` + ```sensor name```. I was thinking of resolving the issue by inserting the name of the iotawatt that the user enters during setup, as long as we can guarantee uniqueness. I haven't gotten that far yet.

I can remove the ```CONF_NAME```, then use the hostname/IP as the sensor name, whenever getting around to fixing that issue? Would the sensor name look ugly with an IP address in it? Or do you think the issue can be resolved without using either ```CONF_NAME``` or the hostname/IP? If so I can remove ```CONF_NAME```

Reply:
Home Assistant allows renaming devices and entities. We shouldn't copy this logic into integrations, so let's drop it .

If someone has 2 IoTaWatt (a tiny percentage I bet) they will be able to go to integration -> entity and rename it as such. So let's drop it.